### DISCUSSION TITLE
Engine DJ 3.0 is now out =)

### DISCUSSION THREAD

**Discussion by jaygarcia** [MarByteBeep/StageLinq#24] [General]
https://www.digitaldjtips.com/engine-dj-3-0-brings-a-sampler-to-standalone-gear-for-the-first-time/

Should be interesting to see how destructive this upgrade is to this library.

**Comment by honusz** [comment]
@jaygarcia I've used it from time to time with the beta, no major issues so far.

They only decided to bump the major version halfway through the beta cycle for this release. By their own reasoning, this was due to the amount of new features included, not because of any breaking changes (hopefully).

**Reply by jaygarcia** [reply]
Effing beautiful!  Many thanks for the quick reply @honusz! Thank you for all of the work you've put into this library. <3 

Is there a patreon (or similar) that I could use to buy you a coffee (or beer?) :D

**Reply by honusz** [reply]
> Effing beautiful!  Many thanks for the quick reply @honusz! Thank you for all of the work you've put into this library. <3 
> 
> Is there a patreon (or similar) that I could use to buy you a coffee (or beer?) :D 

Very kind of you, but honestly, I deserve it least out of the contributors, lol. @MarByteBeep & @icedream have done the yeomans' work on these libs, and @chrisle has done a ton of work keeping it going.

Consider join us on @chrisle 's [discord server](https://discord.gg/zAgUyn7j) where we chat stagelinq stuff, and definitely check out his livestreams and his software [nowplaying2](https://www.nowplaying2.com)

**Reply by jaygarcia** [reply]
Hell, i'd gladly donate. You folks have done pretty amazing work.

I'll join the server =D . Many thanks <3
